pharaoh_examples/lib/api_service/index.dart

```diff
@@ -55,6 +55,17 @@ void main() async {
     next(req);
   }, Route.path('/api'));
 
+  /// Header request
+  /// example: http://localhost:3000/api/users
+  app.head('/api/*', (req, res) => res.status(200).send(''));
+
+  /// Patch request
+  app.patch('/api/user/:name/repos', (req, res) {
+    var name = req.params['name'];
+
+    return res.status(200).send('Updated $name repos');
+  });
+
```

Comment:
This shouldn't be here. When you add a new feature to Pharaoh, you put the tests in the `lib/test` directory.

Reply:
Dropped this change as well